docker/docker-compose.yaml

```
@@ -229,6 +229,14 @@ services:
       RELYT_USER: postgres
       RELYT_PASSWORD: difyai123456
       RELYT_DATABASE: postgres
+      # tencent configurations
+      TENCENT_URL: http://127.0.0.1
```

```suggestion
      TENCENT_VECTOR_DB_URL: http://127.0.0.1
```
To distinguish with other Tencent services. The rest should be the same.

---

resolve